src/main/java/ca/on/oicr/gsi/dimsum/service/CaseService.java

```diff
@@ -601,6 +614,9 @@ private ProjectSummaryRow getCompletedProjectSummaryRow(ProjectSummary projectSu
   }
 
   private ProjectSummaryRow getPendingProjectSummaryRow(ProjectSummary projectSummary) {
+    if (projectSummary == null) {
+      return new ProjectSummaryRow.Builder().title("Pending Work").build();
+    }
```

Comment:
Can be removed as `projectSummary` should never be null. Same in `getPendingQcProjectSummaryRow`